apps/ui/src/components/App/Nav.vue

```diff
@@ -63,91 +72,198 @@ const canSeeSettings = computed(() => {
-            icon: IHLightningBolt
-          }
-        }
-      : undefined),
-    ...(SPACES_DISCUSSIONS[`${networkId.value}:${address.value}`]
-      ? {
-          discussions: {
-            name: 'Discussions',
-            icon: IHAnnotation,
-            active: ['space-discussions', 'space-discussions-topic'].includes(
-              route.name as string
-            )
-          }
-        }
-      : undefined),
-    ...(space.value?.treasuries?.length
-      ? {
-          treasury: {
-            name: 'Treasury',
-            icon: IHCash
-          }
-        }
-      : undefined),
-    ...(canSeeSettings.value
-      ? {
-          settings: {
+function getNavigationConfig(
+  route: RouteLocationNormalizedLoaded
+): NavigationConfig | null {
+  const mainRoute = route.matched[0]?.name;
+
+  function getSettingsRoute({
+    name,
+    tab,
+    hidden
+  }: {
+    name: string;
+    tab: string;
+    hidden?: boolean;
+  }) {
+    return {
+      name,
+      link: {
+        name: 'space-settings',
+        params: { tab }
+      },
+      active: route.params.tab === tab,
+      hidden
+    };
+  }
+
+  if (mainRoute === 'space') {
+    if (route.name === 'space-settings') {
+      const isOffchainNetwork = space.value
+        ? offchainNetworks.includes(space.value.network)
+        : false;
+
+      return {
+        style: 'slim',
+        items: {
+          back: {
             name: 'Settings',
-            icon: IHCog
-          }
+            icon: IHArrowLongLeft,
+            link: { name: 'space-overview' },
+            active: true
+          },
+          profile: getSettingsRoute({
+            name: 'Profile',
+            tab: 'profile'
+          }),
+          proposal: getSettingsRoute({
+            name: 'Proposal',
+            tab: 'proposal'
+          }),
+          votingStrategies: getSettingsRoute({
+            name: 'Voting strategies',
+            tab: 'voting-strategies'
+          }),
+          voting: getSettingsRoute({
+            name: 'Voting',
+            tab: 'voting'
+          }),
+          members: getSettingsRoute({
+            name: 'Members',
+            tab: 'members',
+            hidden: !isOffchainNetwork
+          }),
+          execution: getSettingsRoute({
+            name: 'Execution',
+            tab: 'execution',
+            hidden: isOffchainNetwork
+          }),
+          authenticators: getSettingsRoute({
+            name: 'Authenticators',
+            tab: 'authenticators',
+            hidden: isOffchainNetwork
+          }),
+          treasuries: getSettingsRoute({
+            name: 'Treasuries',
+            tab: 'treasuries'
+          }),
+          delegations: getSettingsRoute({
+            name: 'Delegations',
+            tab: 'delegations'
+          }),
+          labels: getSettingsRoute({
+            name: 'Labels',
+            tab: 'labels'
+          }),
+          whitelabel: getSettingsRoute({
+            name: 'White label',
```

Comment:
Technically it should be "Whitelabel" here, but actually it would be better to rename it to "Custom domain" instead for the users